app/ui/omni-box.js

```diff
@@ -107,6 +109,34 @@ class OmniBox extends HTMLElement {
     this.forwardButton.addEventListener('click', () => {
       this.dispatchEvent(new CustomEvent('forward'))
     })
+
+    // middle mouse click paste&go
+    const win = new BrowserWindow()
```

Comment:
Try not to initialize the window within methods of web components, if you really need it, it should be enough to do once. In this case you don't need BrowserWindow